### PR TITLE
[Reviews] creando modelo de reviews

### PR DESCRIPTION
**Pull request by nestornovella**
DESCRIPTION

se creo el modelo de reviews adjunto imagen de db...

### CODE REVIEW

**Review by agustinhummel — APPROVED** [review]
ok

File: api/database/migrations/20221108225400-Create-Reviews.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+const idCreator = require("../../functions/idCreator.js")
+const { DataTypes }= require("sequelize")
+module.exports = {
+  up: async (queryInterface, Sequelize) => {
+    await queryInterface.createTable("Reviews", {
+      id:{
+          type:DataTypes.STRING,
+          primaryKey: true,
+          allowNull: false,
+          defaultValue: idCreator()
```
Comment (miguelbel00) [inline api/database/migrations/20221108225400-Create-Reviews.js:12]: changes for UUID

**Review by miguelbel00 — CHANGES_REQUESTED** [review]
remove MiguelFile.txt
let .env.example because this is necessary for the new members (in back)
unnecessary idCreator function can be replace for uuid and delete it
on src/models/products why is this folder here?

**Review by iSensative — APPROVED** [review]
All ok

**Review by lucaaramos — APPROVED** [review]
ok

File: api/database/migrations/20221108212203-create-Products.js [hunk 2]
```diff
@@ -0,0 +1,42 @@
+'use strict';
+
+const { DataTypes, UUIDV4 }= require("sequelize")
+module.exports = {
+  up: async (queryInterface, Sequelize) => {
+    await queryInterface.createTable("Products", {
+          id:{
+              type:DataTypes.STRING,
```
Comment (miguelbel00) [inline api/database/migrations/20221108212203-create-Products.js:8]: change DataTypes  for UUID

File: api/database/migrations/20221108225400-Create-Reviews.js [hunk 3]
```diff
@@ -0,0 +1,28 @@
+'use strict';
+const { DataTypes, UUIDV4 }= require("sequelize")
+module.exports = {
+  up: async (queryInterface, Sequelize) => {
+    await queryInterface.createTable("Reviews", {
+      id:{
+          type:DataTypes.STRING,
```
Comment (miguelbel00) [inline api/database/migrations/20221108225400-Create-Reviews.js:7]: Change DataType UUID

**Review by miguelbel00 — CHANGES_REQUESTED** [review]
unnecessary folder src, could be delete

**Review by gabicasas01 — APPROVED** [review]

**Pull request closed without merging**